codec/types/any.go

```diff
@@ -91,7 +91,7 @@ func (any *Any) Pack(x proto.Message) error {
 func UnsafePackAny(x interface{}) *Any {
 	if msg, ok := x.(proto.Message); ok {
 		any, err := NewAnyWithValue(msg)
-		if err != nil {
+		if err == nil {
```

Comment:
😱

Reply:
sigh :(

Reply:
Lol

Reply:
Maybe with the `GenStruct` idea we can remove this function at all (this would require refactoring).